airbyte-integrations/connectors/source-drift/source_drift/spec.json

```diff
@@ -24,8 +24,6 @@
               "credentials": {
                 "type": "string",
                 "const": "oauth2.0",
-                "enum": ["oauth2.0"],
```

Comment:
Are you test this change on UI?

Reply:
I have tested it in UI locally, and the setup works, I removed those lines because CAT wouldn't pass (there was an error message indicating that I should only keep the `const` value in spec)

Reply:
Thanks for checking locally - did you also try turning on buttons in the UI to make sure the fields got mapped correctly to the authenticate with Oauth button? Instructions [here](https://internal-docs.airbyte.io/How-Tos/Test-oAuth-Locally/#turn-on-oauth-buttons-in-the-ui-needs-re-build)